### PR TITLE
Unified tutorial and risk tensor output in all wrappers

### PR DESCRIPTION
**Pull request by egesko**
This PR implements a comprehensive tutorial that touches on all risk metrics, as well as updating all wrapper outputs to be consistent (they now all output risk tensor).

### CODE REVIEW

File: notebooks/0_Unified-Tutorial.ipynb
[the remark this answers is not in the document]
Comment (IaroslavElistratov) [reply notebooks/0_Unified-Tutorial.ipynb:1]: Also, for this tutorial, I believe we want to add @slolla as the commit author. Do you agree with that?

File: notebooks/1_Ensemble-Classification.ipynb
Comment (IaroslavElistratov) [inline notebooks/1_Ensemble-Classification.ipynb:1]: Could we please replace excerpts below with something different
- capsa (a Python Library for Mitigating Uncertainty) --> [something else]
- Uncertainty in a model's predictions due to model limitations --> I think we mean model not being expressive enough to represent the data
- This new ensemble architecture is parallelizable with tensorflow --> I don't think this is correct, we're still looping over each model one by one

File: notebooks/2_Histogram-Classification.ipynb
Comment (IaroslavElistratov) [inline notebooks/2_Histogram-Classification.ipynb:1]: Could we please replace excerpts below with something different
- plot your training dataset into a graph --> plot your training dataset
- requires you to several parameters --> typo
- wrap your model mode --> typo

File: notebooks/0_Unified-Tutorial.ipynb
[the remark this answers is not in the document]
Comment (IaroslavElistratov) [reply notebooks/0_Unified-Tutorial.ipynb:1]: Do you think we can include [this](https://github.com/aamini/introtodeeplearning/blob/master/lab3/Part1_IntroductionCapsa.ipynb) tutorial as well?

[the remark this answers is not in the document]
Comment (aamini) [reply notebooks/0_Unified-Tutorial.ipynb:1]: I agree with [this](https://github.com/themis-ai/capsa/pull/118/files/370b41ba5e12efe5b7d5132de49e2ca91a2a08f8#r1149333238) as well. Would be great to include the Introduction notebook as the one labeled `0_*` and then the current `0_Unified-Tutorial` can be renamed to a later number (as it is more advanced) and titled something about "Faces"

[the remark this answers is not in the document]
Comment (aamini) [reply notebooks/0_Unified-Tutorial.ipynb:1]: I agree with [this](https://github.com/themis-ai/capsa/pull/118/files/370b41ba5e12efe5b7d5132de49e2ca91a2a08f8#r1149333238) as well. Would be great to include the Introduction notebook as the one labeled 0_* and then the current 0_Unified-Tutorial can be renamed to a later number (as it is more advanced) and titled something about "Faces"

File: capsa/bias/histogram.py
[the remark this answers is not in the document]
Comment (aamini) [reply capsa/bias/histogram.py:1]: I'm not sure if this is related but currently it looks like `test_mve.py` is throwing an error in py39 environment?

[the remark this answers is not in the document]
Comment (egesko) [reply capsa/bias/histogram.py:1]: @reza-themisai  it looks like an issue with the docker image being generated, any chance you can look into this?

[the remark this answers is not in the document]
Comment (egesko) [reply capsa/bias/histogram.py:1]: > I understand this is out of the scope for the current PR, but in [92683da](https://github.com/themis-ai/capsa/commit/92683daf7d9538d8d458db888cf8a3a442a13524) it looks you're using `unittest.TestCase` for ensemble and mve, could you please do the same for dropout, histogram and the vae tests? Otherwise our tests right now look a little bit inconsistent. It's perfectly fine if we do it in a separate PR.

I don't have a good test creation framework yet for wrappers since the main thing we test for is whether we are seeing a higher risk metric value than average in certain points in a given dataset when we train using a fixed dataset. We end up having to define what those points should be beforehand, which is fairly counterintuitive. I will generate a tool for this in a separate PR.

[the remark this answers is not in the document]
Comment (egesko) [reply capsa/bias/histogram.py:1]: > I'm not sure if this is related but currently it looks like `test_mve.py` is throwing an error in py39 environment?

#121 addresses this.

**Comment by egesko** [comment]
# Latest updates

Here's what this PR currently does:

- Updates all wrappers to output `RiskTensor()` with corresponding risk metric.
- Updates `get_histogram_probability()` for HistogramWrapper and HistogramVAEWrapper so that it doesn't do softmax over entire batch. Also adds small epsilon to frequencies so that we don't get NaN likelihoods.
- VAEWrapper and HistogramVAEWrapper does not return epistemic risk with extra two dimensions with 1 length anymore.
- Added `[Tutorials]` that points to capsa tutorials page in `README.md`
- Added two tutorials `Introduction` and `Faces`.

File: README.md [hunk 1]
```diff
@@ -98,3 +99,7 @@ Checkout our <a href="https://themisai.io/capsa/tutorials">tutorials</a> on some
 Capsa is being actively maintained and advanced. It has been built with research, extensibility, and community development as a priority. We greatly appreciate contributions to the capsa repository and codebase, including issues, enhancements, and pull requests.
 
 For more details please see <a href="https://themisai.io/capsa/contribute/">here</a>.
+
+# 💡 Support
+
+Capsa currently supports Keras Sequential models, although we are looking at possible solutions for supporting wider range of models. We can't wait to share you some new details soon!
```
Comment (IaroslavElistratov) [inline README.md:105]: Love this

**Review by IaroslavElistratov — APPROVED** [review]
Thank you @egesko!

**Review by aamini — APPROVED** [review]
Love it! Great work @egesko!! 🚀
